modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1640,6 +1640,10 @@
 			<category name="Upgrade" />
 			<description name="Replace the parameter Int for BigDecimal of method validate of 'CommerceOrderValidator' interface" />
 		</check>
+		<check name="UpgradeJavaDDMFormValuesSerializerTrackerCheck">
+			<category name="Upgrade" />
+			<description name="Replaces the references `DDMFormValuesSerializerTracker` class and also its methods usages." />
```

Comment:
@nicolas-sss, I think we should write like `Replaces the references to ...`, I mean include the preposition `to`

Reply:
@natocesarrego Don't should be `Replaces the references of `DDMFormValuesSerializerTracker` class and also its methods usages.`?

Reply:
I did this way, since we are replacing the `DDMFormValuesSerializerTracker`  doesn't make sense use the `to`.